renderers/video_renderer_rpi.c

```diff
@@ -59,36 +59,52 @@ struct video_renderer_s {
     uint64_t input_frames;
 };
 
+DISPMANX_ELEMENT_HANDLE_T background = 0;
```

Comment:
Please move this into the video_renderer_s struct.

Reply:
It has been done in latest commit